### PR TITLE
Support native geth and enable/disable tessera

### PR DESCRIPTION
**Pull request by amalrajmani**
This PR contains changes to :

- support native geth
- enable/disable tessera

### CODE REVIEW

File: stresstest-aws/bootstrap.tf [hunk 1]
```diff
@@ -64,18 +78,37 @@ resource "local_file" "genesis-file" {
...
       "maxCodeSize": 50
     },
     "difficulty": "${var.consensus == "ibft" ? "0x1" : "0x0"}",
-    "extraData": "${var.consensus == "ibft" ? quorum_bootstrap_istanbul_extradata.this.extradata : "0x0000000000000000000000000000000000000000000000000000000000000000"}",
+%{if var.consensus == "ibft"~}
+    "extraData": "${quorum_bootstrap_istanbul_extradata.this.extradata}",
+%{endif~}
+
+%{if var.consensus == "clique"~}
+    "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000${local.clique_extra_data}0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
```
Comment (trung) [inline stresstest-aws/bootstrap.tf:105]: please move the full extra data value to `local.clique_extra_data` variable so we just need to reference here as:
```
...
 "extraData": "${local.clique_extra_data}",
...
```
It provides the better understanding of `local.clique_extra_data`.

File: stresstest-aws/bootstrap.tf [hunk 2]
```diff
@@ -64,18 +78,37 @@ resource "local_file" "genesis-file" {
...
+%{endif~}
+
+
+%{if var.consensus == "raft"~}
+    "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000",
+%{endif~}
```
Comment (trung) [inline stresstest-aws/bootstrap.tf:111]: as we have 3 if conditions, i would declare a local map variable for consensus extra data and retrieve them with key.
```
locals {
   consensus_extra_data = {
     "istanbul" = "${quorum_bootstrap_istanbul_extradata.this.extradata}"
     "clique" = "${local.clique_extra_data}"
   }
}
....

   "extraData": "${lookup(local.consensus_extra_data, var.consensus, "0x0000000000000000000000000000000000000000000000000000000000000000")}"
```

File: stresstest-aws/docker-compose.tf [hunk 3]
```diff
@@ -61,6 +63,11 @@ x-quorum-def:
         --minerthreads 1 \
         --syncmode full \
 %{endif~}
+%{if var.consensus == "clique"~}
+        --mine \
+        --minerthreads 1 \
+        --syncmode full \
+%{endif~}
```
Comment (trung) [inline stresstest-aws/docker-compose.tf:70]: similar solution like recommendation for extraData, we can have `local.consensus_geth_args` map.
Simplify the template with lesser if conditions

File: stresstest-aws/docker-compose.tf [hunk 4]
```diff
@@ -108,18 +117,25 @@ services:
...
     networks:
       ${local.network_name}-net:
         ipv4_address: ${element(data.null_data_source.meta[*].inputs.nodeContainerIP, count.index)}
     environment:
       - ADDITIONAL_GETH_ARGS=${local.geth_addt_args}
+%{if var.enable_tessera == true~}
       - PRIVATE_CONFIG=${local.tm_dir_container_path}/tm.ipc
+%{else~}
+      - PRIVATE_CONFIG=ignore
+%{endif~}
```
Comment (trung) [inline stresstest-aws/docker-compose.tf:133]: can use conditional expression here:
```
PRIVATE_CONFIG=${var.enable_tessera == true ? format("%s/tm.ipc", local.tm_dir_container_path) : "ignore"}
```

File: stresstest-aws/docker-compose.tf (hunk 3 above)
Comment (amalrajmani) [reply stresstest-aws/docker-compose.tf:70]: i tried to use a map to lookup but docker-compose was failing as the code was not getting formatted correctly(left indents were not getting applied though i tried sapce/tabs).

**Pull request merged**